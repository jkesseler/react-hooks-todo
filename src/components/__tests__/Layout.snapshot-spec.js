import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Layout from '../Layout';

describe('Layout', () => {
  it('renders without crashing', () => {
    const component = shallow(<Layout />);
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
