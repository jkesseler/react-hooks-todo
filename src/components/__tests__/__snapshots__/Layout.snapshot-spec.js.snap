// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders without crashing 1`] = `
<WithStyles(ForwardRef(Paper))
  elevation={0}
  style={
    Object {
      "backgroundColor": "#fafafa",
      "margin": 0,
      "padding": 0,
    }
  }
>
  <WithStyles(ForwardRef(AppBar))
    color="primary"
    position="static"
    style={
      Object {
        "height": 64,
      }
    }
  >
    <WithStyles(ForwardRef(Toolbar))
      style={
        Object {
          "height": 60,
        }
      }
    >
      <WithStyles(ForwardRef(Typography))
        color="inherit"
      >
        Todo App
      </WithStyles(ForwardRef(Typography))>
    </WithStyles(ForwardRef(Toolbar))>
  </WithStyles(ForwardRef(AppBar))>
</WithStyles(ForwardRef(Paper))>
`;
